cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -168,43 +181,63 @@ func (o *objectGraph) objToNode(obj *unstructured.Unstructured) *node {
 		virtual:        false,
 	}
 
+	crdStr := crdTypeMetaToString(metav1.TypeMeta{Kind: newNode.identity.Kind, APIVersion: newNode.identity.APIVersion})
+	if _, ok := o.crds[crdStr]; ok {
+		if o.crds[crdStr].moveLabel {
+			newNode.moveLabel = true
+		}
+	}
```

Comment:
The same test should be implemented in ownerToVirtualNode, so I suggest to refactor in a func so we can have something like
```
forceMove: getForceMove(APIversion, kind)
```
in both functions